storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/SnowflakeQueryTemplater.java

```diff
@@ -70,30 +70,34 @@ public SnowflakeQueryTemplater(JdbcConnectionProvider connectionProvider) {
     return createEntityTableRowCountQueries;
   }
 
+  // TODO: support filepath as s3 location
   @Override
   protected List<String> createLoadEntityQuery(
-      String destinationTable, String temporaryTable, File filePath) {
+      String destinationTable, String stagingLocation, URI fileUri) {
```

Comment:
current implementation will only support s3 entity tables under the same staging location.

Reply:
As discussed, the entity and staging locations should be uncoupled